integration_tests/test_snap.py

```diff
@@ -169,3 +169,17 @@ def test_error_on_bad_yaml(self):
             "Issues while validating snapcraft.yaml: found character '\\t' "
             "that cannot start any token on line 13 of snapcraft.yaml",
             str(error.output))
+
+    def test_prime_includes_stage_fileset(self):
+        project_dir = 'prime-from-stage'
```

Comment:
This project is nowhere. Missing add?
Also, I think this should be in test_prime.py